sqlserver/requirements.in

```diff
@@ -1,5 +1,5 @@
 adodbapi==2.6.0.7; sys_platform == 'win32'
-pyodbc==4.0.13; sys_platform == 'win32'
+pyodbc==4.0.13
```

Comment:
You should compile the `.txt` for this to take effect

Reply:
Missed that, thanks.